addons/mail/static/src/discuss/call/common/call_actions.js

```diff
@@ -206,9 +206,7 @@ registerCallAction("join-with-camera", {
 });
 registerCallAction("join", {
     condition: ({ store, thread }) =>
-        !thread?.eq(store.rtc?.channel) &&
-        !thread?.self_member_id?.rtc_inviting_session_id &&
-        typeof thread?.useCameraByDefault !== "boolean",
```

Comment:
It's always a bad sign that you are testing `typeof` instead of an actual value.

Reply:
I see, but I see an indication that we are using a bad design somewhere and we should clean it at some point